infrastructure/eth-networks/private-testnet/.gitignore

```diff
@@ -0,0 +1,13 @@
+# Local NPM Command Installation
+package.json
+package-lock.json
+node_modules/
+
+# Secrets
+bundles/*/secret/
```

Comment:
What about `bundles/*/config`?

Reply:
It's just the encrypted key file. In V1 we were committing them. May be good for tracking.

Reply:
OK, this will help us tracking who is who on testnet.